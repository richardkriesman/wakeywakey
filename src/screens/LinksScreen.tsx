import { ExpoLinksView } from "@expo/samples";
import React from "react";
import { ScrollView, StyleSheet } from "react-native";

export default class LinksScreen extends React.Component {
    public static navigationOptions = {
        title: "Links",
    };

    public render() {
        return (
            <ScrollView style={styles.container}>
                {/* Go ahead and delete ExpoLinksView and replace it with your
           * content, we just wanted to provide you with some helpful links */}
                <ExpoLinksView />
            </ScrollView>
        );
    }
}

const styles = StyleSheet.create({
    container: {
        backgroundColor: "#fff",
        flex: 1,
        paddingTop: 15
    },
});
