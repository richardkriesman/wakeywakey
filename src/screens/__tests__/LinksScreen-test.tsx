import React from "react";
import "react-native";
import renderer from "react-test-renderer";
import LinksScreen from "../LinksScreen";

describe("LinksScreen snapshot", () => {

    it("renders correctly", () => {
        const tree = renderer.create(<LinksScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });

});
