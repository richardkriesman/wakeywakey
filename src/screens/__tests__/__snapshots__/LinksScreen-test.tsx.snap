// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinksScreen snapshot renders correctly 1`] = `null`;
